### PR TITLE
Adding size limits to probo assets documentation.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: docs/build-configuration/assets.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ permalink: /build/assets/
 redirect_from: /assets/
 published: true
 ---
```
Comment (jkaeser) [inline docs/build-configuration/assets.md:8]: > For example, you may need to upload a database dump for your project, or a .ENV file to hold some API credentials.

Technically you shouldn't have the second comma in this sentence.

**Review by jkaeser — CHANGES_REQUESTED** [review]
@sk33lz One nitpick!

File: docs/build-configuration/assets.md (hunk 1 above)
Comment (sk33lz) [reply docs/build-configuration/assets.md:8]: Get out the ruler!

**Review by jkaeser — APPROVED** [review]
+1

**Pull request merged**